enforce-signer.tf

```diff
@@ -13,15 +13,22 @@ resource "google_service_account_iam_member" "allow_signer_impersonation" {
   member             = "principalSet://iam.googleapis.com/${google_iam_workload_identity_pool.chainguard_pool.name}/attribute.sub/enforce-signer:${var.enforce_group_id}"
 }
 
-resource "google_project_iam_member" "signer_cert_requester" {
-  project = local.project_id
-  role    = "roles/privateca.certificateRequester"
-  member  = "serviceAccount:${google_service_account.chainguard_signer.email}"
+// TODO(hectorj2f) Think about restricting cloudkms.cryptoKeyEncrypterDecrypter permissions to a keyring, e.g. key_ring_id = google_kms_key_ring.sigstore-keyring.id
+resource "google_project_iam_custom_role" "chainguard_signer_ca_role" {
+  project     = local.project_id
+  role_id     = "chainguardSignerCA"
+  title       = "Custom Role Chainguard Signer CA"
+  description = "Chainguard signer role to list certificates from a CA"
+  permissions = [
+    "privateca.certificateAuthorities.list",
+    "privateca.certificateAuthorities.get",
+    "privateca.certificates.create",
+    "cloudkms.cryptoKeyEncrypterDecrypter",
```

Comment:
Unfortunately this is a role, which is incompatible with this context so the latest module release is broken.

I think @tcnghia may have a fix for this, but I just hit it as well 😞

cc @priyawadhwa

Reply:
sorry, i had meant to remove this in #12 but must have forgotten. opened up https://github.com/chainguard-dev/terraform-google-chainguard-account-association/pull/14 as a fix!

Reply:
yes, sorry. I added it by mistake to this list here 😞 .